src/prefect/client/orchestration.py

```diff
@@ -206,6 +208,11 @@ def __init__(
 
         if PREFECT_API_TLS_INSECURE_SKIP_VERIFY:
             httpx_settings.setdefault("verify", False)
+        else:
+            cert_file = os.environ.get("SSL_CERT_FILE")
```

Comment:
What do you think about making this a Prefect setting? I'm thinking that it's possible that someone would want Prefect configured differently than the OpenSSL env var. It would look something like `PREFECT_SSL_CERT_FILE = Setting(Path, default=os.environ.get("SSL_CERT_FILE"))` and then you'd use it here with `settings.PREFECT_SSL_CERT_FILE.value()`

Reply:
I have pushed a new commit that added the new setting `PREFECT_SSL_CERT_FILE`. Please review my code again.